docs/conf/prague/2020/schedule.rst

```diff
@@ -169,7 +187,7 @@ Job Fair
 We'll be holding a job fair on Tuesday morning!
 
 * **Where**: {{about.venue}}, {{about.unconfroom}}
-* **When**: **10:30-11:50**
+* **When**: **10:30-11:50 {{ tz }}**
```

Comment:
We should backport this change to Portland if we haven't already 👍

Reply:
It doesn't help us with 2:00 PM / 14:00 issue though. Is there an 'easy' format_this_according_to_conf_location function we can use?

Reply:
Not that I know of off hand.

Reply:
Going to resolve it for this PR and throw a card in the Global Trello board to handle timezones